lib/request-handler.js

```diff
@@ -156,7 +156,7 @@ module.exports = function processRequest(options, request, response) {
                 request.headers
             );
 
-            const configAssets = await getAssetsToPreload();
+            const configAssets = await getAssetsToPreload(request);
```

Comment:
Pls write a test that will check that we pass request object into getAssetsToPreload callback

Reply:
Covered